configure.ac

```diff
@@ -6,8 +6,9 @@
 #
 
 # Initialization
-AC_INIT([aws-ofi-nccl], [GitHub-dev], [al-ofi-nccl-team@amazon.com], , [http://github.com/aws/aws-ofi-nccl])
+AC_INIT([aws-ofi-nccl],[GitHub-dev],[al-ofi-nccl-team@amazon.com],[],[http://github.com/aws/aws-ofi-nccl])
```

Comment:
this is not mentioned in your commit message.

Reply:
autoupdate did this to me and it snuck into my diff. Cleaned up for repost, thanks.